### DISCUSSION TITLE
Help with quickfix provider

### DISCUSSION THREAD

**Discussion by GSri30** [eclipse-xtext/xtext#2746] [Q&A] [answered]
I was going through the quickfix provider code. I'm using xtext 2.26.

I also went through few of the existing implementations like
https://github.com/xtext/xtext-languageserver-example/blob/80df168e906e6a36b937f698418017c47b30cb70/org.xtext.example.mydsl.ide/src/org/xtext/example/mydsl/ide/MyDslIdeQuickfixProvider.java

The problem I'm facing is that, in 2.26, the `createTextEdit` function is not available. I couldn't find any way to get `IModification<EObject> modification`, to pass it as second argument for `.accept` function.

The following is the code I have

```
public class CustomQuickFixProvider extends AbstractDeclarativeIdeQuickfixProvider {
    
    @QuickFix(TSLValidator.INVALID_GUID)
    public void textFixLowerCaseName(DiagnosticResolutionAcceptor acceptor) {
        modification = ?
        acceptor.accept("Capitalize Name", modification
            
        );
    }
}
```

<img width="878" alt="Screenshot 2023-07-17 at 3 31 09 AM" src="https://github.com/eclipse/xtext/assets/54752266/0c047f55-4e63-443f-820d-82b35dd32cf9">

**Comment by cdietrich** [comment, marked as answer]
please check the history https://github.com/xtext/xtext-languageserver-example/commit/835129f3fb9749d4e0e662986b69eebed1015a58 (or older) also check package.json
